backend/src/socket/ndauConnect.js

```diff
@@ -26,7 +26,7 @@ module.exports = (_io) => {
     console.log(socket.id);
```

Comment:
Please just rename the file name ndauConnect to, says, walletConnect or something similar